geometry_msgs/Point32[] leftLine
geometry_msgs/Point32[] centreLine
geometry_msgs/Point32[] rightLine